News/Controllers/ArticleController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using News.Entities.Models;
+using News.Services.ServicesInterface;
+
+namespace News.Controllers;
+
+[ApiController]
+[Route("article")]
+public class ArticleController : ControllerBase
+{    
+    private readonly IArticleService _service;
+    public ArticleController(DbContext context, IArticleService service)
+    {        
+        _service = service;
+    }
+
+    [HttpGet]
+    public IActionResult GetById([FromQuery] Guid id)
```

Comment:
Get id from Route. You can check an example in UsersController.

Reply:
done